.github/workflows/tests.yml

```diff
@@ -4,8 +4,9 @@ on:
   pull_request:
     branches:
       - main
-    paths-ignore:
-      - website/**
+    paths:
```

Comment:
don't we want CI to run again if project deps changes? (ie lockfile)

Reply:
I thought about it, but I can't think of a case where lockfile changes without package.json changes as well. Because if package.json doesn't change, there will be no difference to our user anyways.

Reply:
You can upgrade Yarn package resolutions and yet keep package.json intact, as long as semver is respected

It's not what we do in practice when we upgrade deps so we can give it a try like that and see if it causes any problem 👍